src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
-body {
+$cardWidth: 198px;
+$borderStyle: 1px solid #f3f3f3;
+$linkBlueColor: #00acdc;
+
+@mixin reset-default-styles {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+@mixin font {
+  font-family: "Roboto", "Arial", sans-serif;
+  font-style: normal;
+}
+
+body {
+  @include reset-default-styles();
+  @include font();
+}
+
+.card {
+  border: $borderStyle;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $cardWidth;
+
+  &__img {
+    margin: 32px 19px 40px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 16px 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__description {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-left: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 16px 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 16px 16px;
+  }
+
+  &__link {
+    margin: 0 16px 16px;
+    padding: 12px 0;
+    background: $linkBlueColor;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+  }
+
+  &__link:hover {
+    padding: 11px 0;
+    border: 1px solid $linkBlueColor;
+    color: $linkBlueColor;
+    background-color: #fff;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    background-position: center;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
read requirements:
`Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.`